src/components/SubmitButton/SubmitButton.styles.ts

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import { transparentize } from 'polished';
+import { colors } from '@keen.io/colors';
+
+export const SuccessButton = styled.button`
+  background-color: ${colors.green['500']};
+  border: solid 1px ${colors.green['300']};
+  color: ${colors.white['500']};
+  border-radius: 0px;
+  height: 47px;
+  display: inline-block;
+  width: 125px;
+  font-size: 14px;
+  line-height: 17px;
+  box-shadow: 0 2px 4px 0 ${transparentize(0.85, colors.black['500'])};
+  &:hover {
+    box-shadow: none;
+    color: ${colors.white['500']};
+    background-color: ${colors.green['300']};
+  }
+  letter-spacing: 0;
+  font-family: 'Lato', sans-serif;
```

Comment:
please use `Lato Regular`

as we're using fonts from `https://static.keen.io/assets/keen-fonts.css`